lib/ohai/plugins/habitat.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+#
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+Ohai.plugin(:Habitat) do
+  provides "habitat"
+
+  def habitat_binary
+    which("hab")
```

Comment:
This avoids having to find it in the path multiple times during this plugin run

```suggestion
    @hab ||= which("hab")
```